department-of-veterans-affairs/vets-api
# POA v2: Add config to disable POA jobs

## Summary

We need the ability to disable the POA jobs in Sandbox. This avoids complex mocking of BGS services since all of the BGS writes happen inside of the jobs that we enqueue from the POA controller.

- Adds a config, `Settings.claims_api.poa_v2.disable_jobs` to `settings.yml`
- Code is setup so that by default jobs would run even if config isn't present

## Related issue(s)

- [API-33689](https://jira.devops.va.gov/browse/API-33689)

## Testing done

- Set `Settings.claims_api.poa_v2.disable_jobs` to `true` in `settings.yml`
- Create a POST to /2122
- Check that the job isn't enqueued
- Go through the same procedure but set the flag to `false`, job should now be queued up

## What areas of the site does it impact?
Lighthouse Benefits Claims API POA v2

## Acceptance criteria
